scripts/install

```diff
@@ -104,159 +48,61 @@ ynh_app_setting_set --app=$app --key=virtualisation --value=$virtualisation
 #=================================================
 ynh_script_progression --message="Storing installation settings..."  --weight=1
 
-ynh_app_setting_set --app=$app --key=domain --value=$domain
 ynh_app_setting_set --app=$app --key=rpc_secret --value=$rpc_secret
-ynh_app_setting_set --app=$app --key=datadir --value=$datadir
 ynh_app_setting_set --app=$app --key=bootstrap_peers --value=$bootstrap_peers
 ynh_app_setting_set --app=$app --key=weight --value=$weight
```

Comment:
I'd move this to the block where rpc_secret / bootstrap_peers are validated/initialized

Also not sure that `weight` does need to be overwritten, it should already be saved like other manifest-provided args